sdk/cosmos/azure-cosmos/docs/StatusCodes.md

```diff
@@ -0,0 +1,53 @@
+# Cosmos DB Java SDK - Status Codes
+
+
+
+## Overview
+
+Below is a summary of the different status code / sub status code combinations that developers can experience in their application when using the Cosmos DB Java SDK. The column "Always transient" indicates whether this error condition is to be expected to always be transient (and auto-mitigated within seconds or at least few minutes). For error conditions that are not always transient the description contains additional context on what to watch out for if this error is seen for longer period of times. For errors that are supposed to be "always transient" it will be necessary to file a support ticket if the error condition exists for an extended period of time. The "Additional Info" column will also point to known issues if available and possible mitigations.
```

Comment:
> can experience in their application

We are including combinations that are handled by the SDK internally and do not surface to the customer though. Should we remove those cases from the list? Or convert them to the actual status/substatus that the app level will receive (like the case of 410/21007)?